docs/boundary-services-a-to-z.adoc

```diff
@@ -63,29 +64,42 @@ Using 10.85.0.1 as gateway ip
 + GATEWAY_MAC=68:d7:9a:1f:77:a1
 + echo 'Using 68:d7:9a:1f:77:a1 as gateway mac'
 Using 68:d7:9a:1f:77:a1 as gateway mac
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' port add 1:0 100G RS
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' port add 2:0 100G RS
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' addr add 1:0 fe80::aae1:deff:fe01:701c
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' addr add 2:0 fe80::aae1:deff:fe01:701d
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' addr add 1:0 fd00:99::1
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' route add fd00:1122:3344:0101::/64 1:0 fe80::aae1:deff:fe00:1
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' arp add fe80::aae1:deff:fe00:1 a8:e1:de:00:00:01
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' route add 0.0.0.0/0 2:0 10.85.0.1
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' arp add 10.85.0.1 68:d7:9a:1f:77:a1
-+ ./out/softnpu/swadm -h '[fd00:1122:3344:101::2]' port list
++ z_swadm port create 1:0 100G RS
```

Comment:
Not sure where the `swadm` binary in `z_swadm` comes from, but if it's recent enough we should probably move to the link-based API. This call would be `z_swadm link create rear0 --fec rs --speed 100G`, for example.

Reply:
This is the binary that is packaged in the `switch` zone, during the `image` buildomat job for `dendrite`. I essentially cherry picked tools/scrimlet/softnpu-init.sh from #2645.